tests/tests.cc

```diff
@@ -72,4 +84,22 @@ TEST_CASE("BFS picks the best path", "BFS") {
   WikiGraph w("./datasets/diff_paths.tsv");
   REQUIRE(expectedAE == w.getPathBFS("A", "E"));
   REQUIRE(expectedAD == w.getPathBFS("A", "D"));
+}
+
+TEST_CASE("BFS handles cycles", "BFS") {
+  std::vector<std::string> expectedAE = {
+    "A", "E"
+  };
+  std::vector<std::string> expectedAD = {
+    "A", "B", "D"
+  };
+  WikiGraph w("./datasets/cycle.tsv");
+  REQUIRE(expectedAE == w.getPathBFS("A", "E"));
+  REQUIRE(expectedAD == w.getPathBFS("A", "D"));
+}
+
+TEST_CASE("BFS matches Dijkstra's given real data", "real") {
+  WikiGraph w("./datasets/links.tsv");
+
+  REQUIRE(w.getPathBFS("12th century", "Walrus") == w.getPathDijkstras("12th century", "Walrus"));
```

Comment:
Good so far. I think we can add some more testing on the real data set. We'll have to be a bit more thorough for the demo. I